benchmarks/scripts/compare.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

This is more portable, especially for *BSD variant